xml/depl_nodes.xml

```diff
@@ -5002,6 +5008,461 @@ getent group postgres | cut -d: -f3</screen>
+    </para>
+
+  <figure>
+   <title>The &o_monitor; &barcl; Raw Mode</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="depl_barclamp_monasca_raw.png" width="100%" format="png"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="depl_barclamp_monasca_raw.png" width="75%" format="png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+    <bridgehead renderas="sect3"><guimenu>agent: settings for openstack-monasca-agent</guimenu></bridgehead>
+
+    <variablelist>
+      <varlistentry>
+      <term>keystone</term>
+      <listitem>
+        <para>
+          Contains &o_ident; credentials that the agents use to send
+	  metrics. Do not change the this options, as it is configured by &crow;.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>insecure</term>
+      <listitem>
+        <para>
+          Specifies whether SSL certificates are verified when
+	  communicating with &o_ident;. If set to <literal>false</literal>, the
+	  <literal>ca_file</literal> option must be disabled as well.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>ca_file</term>
+      <listitem>
+        <para>
+          Specifies whether a CA certificate is used for verifying &o_ident;'s SSL certificate.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>log_dir</term>
+      <listitem>
+        <para>
+          Path for storing log files. The specified path must exist. Do not change the default
+	  <filename>/var/log/monasca-agent</filename> path.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>log_level</term>
+      <listitem>
+        <para>
+          Agent's log level. Limits log messages to the specified level and
+	  above. The following levels are available: Error, Warning, Info (default), and Debug.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>check_frequency</term>
+      <listitem>
+        <para>
+          Interval in seconds between running agents' checks.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>num_collector_threads</term>
+      <listitem>
+        <para>
+          Number of simultaneous collector threads to run. This refers to the
+	  maximum number of different collector plug-ins (for example,
+	  <literal>http_check</literal>) that are allowed run simultaneously. The default
+	  value <literal>1</literal> means that plug-ins are run sequentially.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>pool_full_max_retries</term>
+      <listitem>
+        <para>
+	  If a problem with multiple plug-ins results is blocking the entire
```

Comment:
you mean *a problem with multiple plug-ins results*? In this case, I'd prefer to rewrite this as *a problem with the results from multiple plug-ins*. Does this make sense?